new.html

```diff
@@ -50,43 +202,117 @@
 					this.el.dispatchEvent(event);
 				}
 			},
-			init: function (event) {	
-				console.log('init')			
+			init: function (event) {
+				console.log('init')
 				this.el = event.currentTarget;
 				this.updateUrl();
 				var state = this.reflectState();
-			
+
 				if (!state.q) {
-					state.q = window.btoa(JSON.stringify(config||{}));
+					state.q = window.btoa(JSON.stringify(config || {}));
 				}
 				this.updateState(state);
 			},
-			onpopstate: function(event) {
+			onpopstate: function (event) {
 				this.updateState(event.state);
-				console.log('popstate', event, this.state);				
+				console.log('popstate', event, this.state);
 			},
-			onstatechange: function() {
-				this.reflectStateInUrl(this.sate)
+			onstatechange: function () {
+				this.reflectStateInUrl(this.sate);
```

Comment:
this.sate typo :)

Reply:
Oh no. I wonder would it even work after the fix. Good catch!